llvm/lib/Target/RISCV/RISCVInstrInfoZvk.td

```diff
@@ -88,7 +85,8 @@ class PALUVINoVm<bits<6> funct6, string opcodestr, Operand optype>
 class PALUVINoVmBinary<bits<6> funct6, string opcodestr, Operand optype>
     : RVInstIVI<funct6, (outs VR:$vd_wb),
                 (ins VR:$vd, VR:$vs2, optype:$imm),
-                opcodestr, "$vd, $vs2, $imm"> {
+                opcodestr, "$vd, $vs2, $imm">,
+      SchedBinaryMC<"WriteVIALUV", "ReadVIALUV", "ReadVIALUV"> {
```

Comment:
The same reasoning applies to this one and `PALUVs2NoVmBinary`. Sorry I didn't say it before.

Reply:
PALUVs2NoVmBinary was not addressed.